core/sql/exp/exp_math_func.cpp

```diff
@@ -337,8 +338,22 @@ ex_expr::exp_return_type ExFunctionMath::eval(char *op_data[],
 	  **diagsArea << DgString0("LOG");
 	  return ex_expr::EXPR_ERROR;
 	}
-      
-      *(double *)op_data[0] = MathLog(*(double *)op_data[1], err);
+
+      if(3 == getNumOperands())
+    {
```

Comment:
The indentation is inconsistent. Perhaps your editor is using tabs? Best to replace those with spaces.